src/pages/favorites-page/favorites-page.tsx

```diff
@@ -1,13 +1,17 @@
+import Logo from '../../components/logo/logo';
+import { Helmet } from 'react-helmet-async';
+
 function FavoritesPage() {
   return (
     <div className="page">
+      <Helmet>
+        <title>{'6 cities - Favorites'}</title>
```

Comment:
Экранирование здесь не нужно
```suggestion
        <title>6 cities - Favorites</title>
```